engines/director/frame.h

```diff
@@ -88,6 +88,83 @@ struct PaletteInfo {
 	}
 };
 
+struct MainChannels {
+	CastMemberID _actionId;
```

Comment:
For easiness and per recommendation in our code naming conventions, drop `_` in this struct members.

In short, replace `_mainChannels._` -> `_mainChannels.`, so `_mainChannels._tempo` becomes `_mainChannels.tempo`